lessons/operator/01/README.md

```diff
@@ -1,122 +1,121 @@
-# Lesson 1: Introduction to Monitoring as Code
+# Lesson 1: Introduction to Sensu
 
-- [What is Monitoring as Code?](#what-is-monitoring-as-code)
-- [What is an Observability Pipeline?](#what-is-an-observability-pipeline)
-- [Why Sensu?](#why-sensu)
-- [Use cases](#use-cases)
-- [Architecture](#architecture)
-- [Learn more](#learn-more)
-- [Next steps](#next-steps)
+- [Goals](#goals)
+- [Observability Pipeline](#observability-pipeline)
+- [How Does Sensu Work?](#how-does-sensu-work)
+  - [Sensu Data Model](#sensu-data-model)
+- [Monitoring As Code](#monitoring-as-code)
+  - [Sensu is an API, Configurations are YAML](#sensu-is-an-api-configurations-are-yaml)
+  - [Bonsai Asset Repository](#bonsai-asset-repository)
+- [Discussion](#discussion)
+- [Learn More](#learn-more)
+- [Next Steps](#next-steps)
 
-## What is Monitoring as Code?
+## Goals
+In this lesson we will introduce the Sensu observability pipeline. You will learn about Sensu's architecture, its underlying data model, and how its unique set of features can be used to implement a monitoring-as-code solution.
```

Comment:
s/unique set of features/industry-leading monitoring as code solution/ ??? 

s/can be used to implement a monitoring-as-code solution/can be leveraged to improve infrastructure and application monitoring/ ??? 

Perhaps not these exact words, but "monitoring-as-code" is the "how" not the "what".

Reply:
switched to: "unique set of features can be used to implement a monitoring-as-code workflow within your organization" from "how its unique set of features can be used to implement a monitoring-as-code solution"... which i think subtly emphasizes the distinction you're calling out here without too many buzzwords.